From 79a7cc0e725c4a08e26d0094ad0724be06f659af Mon Sep 17 00:00:00 2001
From: dgibson <dgibson@redhat.com>
Date: Wed, 5 Nov 2014 07:07:42 +0100
Subject: [PATCH 11/15] Downstream only: Remove assorted unneeded devices for
 POWER

Message-id: <1415171263-1587-3-git-send-email-dgibson@redhat.com>
Patchwork-id: 62136
O-Subject: [RHEV-7.1 qemu-kvm-rhev PATCHv3 2/3] Downstream only: Remove assorted unneeded devices for POWER
Bugzilla: 1160120
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

This adjusts the ppc64 configuration to remove devices which aren't
necessasry (and we don't wish to support) for KVM guests on POWER.

Specifically removed are: PC keyboard controller, floppy disk controller,
the "pflash" devices and the "ptimer".

Signed-off-by: David Gibson <dgibson@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/ppc64-softmmu.mak | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/default-configs/ppc64-softmmu.mak b/default-configs/ppc64-softmmu.mak
index bd83712..9a9f40e 100644
--- a/default-configs/ppc64-softmmu.mak
+++ b/default-configs/ppc64-softmmu.mak
@@ -16,15 +16,10 @@ CONFIG_VGA=y
 CONFIG_VGA_PCI=y
 CONFIG_SERIAL=y
 CONFIG_I8254=y
-CONFIG_PCKBD=y
-CONFIG_FDC=y
 CONFIG_I8257=y
 CONFIG_I82374=y
 CONFIG_PC87312=y
 CONFIG_PCSPK=y
-CONFIG_PFLASH_CFI01=y
-CONFIG_PFLASH_CFI02=y
-CONFIG_PTIMER=y
 CONFIG_I8259=y
 CONFIG_PSERIES=y
 # For pSeries
-- 
1.8.3.1

